notes.md

```diff
@@ -49,9 +49,9 @@ Encryption information is stored in the `ContentEncodings Element` under the `Co
 
 # Image cropping
 
-Thanks to the PixelCropXXX elements, it's possible to crop the image before being resized. That means the image size follows this path:
+Thanks to the `PixelCrop Elements` (`PixelCropTop`, `PixelCropBottom`, `PixelCropRight` and `PixelCropLeft`), it's possible to crop the image before being resized. The image size follows this pattern, using `PixelCropTop` as an example:
```

Comment:
> …it's possible to crop the image before being resized.

This sounds strange to me (not only in your modification but in the original already). I'd prefer either "it's possible to crop the image before resizing it" or "it's possible to crop the image before it's resized".